Amazon.IonObjectMapper.Test/Utils.cs

```diff
@@ -114,17 +114,27 @@ public static T DeserializeWithCustomSerializer<T>(IonSerializer<T> customSerial
         public static IIonValue SerializeToIonWithCustomSerializers<T>(
             Dictionary<Type, IIonSerializer> ionSerializers, T item)
         {
-            var serializer = new IonSerializer(new IonSerializationOptions { IonSerializers = ionSerializers });
+            Dictionary<string, object> customContext = new Dictionary<string, object>() 
```

Comment:
Is this custom context needed for these tests? (note that these tests are for custom serializers defined in the `IonSerializers` option which is different than the custom serializer/factory attributes).

Reply:
Made this change as I was using the same `Engine` object both for custom serializers and custom context. Addressed this by creating a new test object for custom context.